lib/timeline/component/css/item.css

```diff
@@ -19,7 +19,7 @@
 }
 
 .vis-editable.vis-selected {
```

Comment:
Why not overriding this class in golive ? Because it's really specific to our needs and maybe, should not be included in vis-timeline if we expect our changes to be backported to original vis-timeline project.

Reply:
j’y ai pensé mais pas sûr que le comportement soit le même avec l’override.je ne pense pas que les classes seront évaluées ds le même ordre.D’autre part, du moment qu’on ajoute plus de finesse (dates début / fin éditables/pas éditables) je penses que de mettre un move cursor sans tenir compte des actions réellement autorisées ne fait plus de sens à mon avis.